src/main/java/com/igot/cb/community/controller/CommunityController.java

```diff
@@ -45,4 +46,24 @@ public ResponseEntity<ApiResponse> update(@RequestBody JsonNode communityDetails
         ApiResponse response = communityManagementService.update(communityDetails, authToken);
         return new ResponseEntity<>(response, response.getResponseCode());
     }
+
+    @PutMapping("/join")
+    public ResponseEntity<ApiResponse> join(@RequestBody Map<String, Object> request,
```

Comment:
this implementation is not correct.... 
UI will have the details while loading that the logged in user is joined or not joined... 
so it has to be join operation only and do define one more endpoint to leave from community... 
you can have service method which takes param -- just pass it is add operation or leave operation

Reply:
Made change to handle join and un-join in separate API